.eslintrc.json

```diff
@@ -1,21 +1,199 @@
 {
-  "extends": "standard",
+  "parserOptions": {
+    // Node 14 already supports ES2020, Node 12 only supports ES2019
+    // https://node.green/#ES2020
+    "ecmaVersion": 2020,
```

Comment:
This should be save as we still run webpack with the ES2015 config.

Reply:
Should we force the people to Node 14? For me it's fine but can you please add this to the readme in the development section.

Reply:
This is just the tooling, not the final build. there's also an .nvmrc file.
And the comment is only about supported features in the eslint parsing, not in the environment itself. Sorry, didn't mention that.